### PR TITLE
docs: More details in client library help

### PR DESCRIPTION
**Pull request by jskeet**
(These were intended to be part of an earlier commit, but hadn't been saved...)

### CODE REVIEW

File: docs/devsite-help/breaking-gax4.md [hunk 1]
```diff
@@ -35,3 +35,24 @@ Some libraries (e.g. PubSub) included IAM methods directly within the client sur
 These are now exposed via a separate client, accessed via the `IAMPolicyClient` property
 of the "main" API client. For example, a previous call of `client.GetIamPolicy(name)`
 should be changed to `client.IAMPolicyClient.GetIamPolicy(name)`.
+
+## Client implementation constructors
+
+The gRPC-based APIs have always split the client classes into an abstract base class (e.g. `ExampleClient`)
```
Comment (amanda-tarafa) [inline docs/devsite-help/breaking-gax4.md:41]:
```suggestion
The client libraries for the gRPC-based APIs have always split the client classes into an abstract base class (e.g. `ExampleClient`)
```

File: docs/devsite-help/breaking-gax4.md [hunk 2]
```diff
@@ -35,3 +35,24 @@ Some libraries (e.g. PubSub) included IAM methods directly within the client sur
...
+```csharp
+public ExampleClientImpl(Example.ExampleClient grpcClient, ExampleSettings settings)
+```
+
+(`Example.ExampleClient` is the generated low-level gRPC stub.)
+
+In GAX v4, the constructor also accepts a logger:
+
+```csharp
+public ExampleClientImpl(Example.ExampleClient grpcClient, ExampleSettings settings)
```
Comment (amanda-tarafa) [inline docs/devsite-help/breaking-gax4.md:57]: The logger is not shown here :)

**Review by amanda-tarafa — APPROVED** [review]
LGTM
But one change needed for sure ;)

File: docs/devsite-help/breaking-gax4.md (hunk 1 above)
Comment (jskeet) [reply docs/devsite-help/breaking-gax4.md:41]: Done, thanks.

File: docs/devsite-help/breaking-gax4.md (hunk 2 above)
Comment (jskeet) [reply docs/devsite-help/breaking-gax4.md:57]: Oops! Fixed, thanks.

Comment (jskeet) [reply docs/devsite-help/breaking-gax4.md]: Doh!

**Pull request merged**